feature_tests/steps/_1_setup.py

```diff
@@ -84,6 +84,21 @@ def has_permissions(
     test_config.request.headers[CONNECTION_METADATA] = json.dumps(existing_headers)
 
 
+@given('{actor_type} "{actor}" has the permissions')
+def has_permissions(
+    context: Context,
+    actor_type: str,
+    actor: str,
+):
+    test_config: TestConfig = context.test_config
+    existing_headers = json_loads(test_config.request.headers[CONNECTION_METADATA])
+
+    permissions = [f'{row["permission"]}' for row in context.table]
```

Comment:
```suggestion
    permissions = [row["permission"] for row in context.table]
```

unless this is to convert to string, in which case the explicit method is more direct:

```suggestion
    permissions = [str(row["permission"]) for row in context.table]
```